src/sentry/snuba/events.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+
+from enum import Enum
+from collections import namedtuple
+
+Column = namedtuple("Column", "event_name transaction_name discover_name alias")
+
+
+class Columns(Enum):
+    """
+    Value is a tuple of (internal Events name, internal Transaction name, internal
+    Discover name, external alias)
+    None means the column is not available in that dataset.
+    """
+
+    EVENT_ID = Column("event_id", "event_id", "event_id", "id")
+    GROUP_ID = Column("group_id", None, "group_id", "issue.id")
+    ISSUE = Column("issue", None, "group_id", "issue.id")
+    PROJECT_ID = Column("project_id", "project_id", "project_id", "project.id")
+    TIMESTAMP = Column("timestamp", "finish_ts", "timestamp", "timestamp")
+    TIME = Column("time", "bucketed_end", "time", "time")
+    CULPRIT = Column("culprit", None, "culprit", "culprit")
+    LOCATION = Column("location", None, "location", "location")
+    MESSAGE = Column("message", "transaction_name", "message", "message")
+    PLATFORM = Column("platform", "platform", "platform", "platform.name")
+    ENVIRONMENT = Column("environment", "environment", "environment", "environment")
+    RELEASE = Column("tags[sentry:release]", "release", "sentry:release", "release")
```

Comment:
Isn't the discover name going to be `release`?

Reply:
Thanks, made the same fix as https://github.com/getsentry/sentry/pull/15616